docs/src/examples/components/Divider/Variations/DividerExampleVertical.shorthand.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import { Divider, Button, Avatar, Text } from '@fluentui/react'
+
+const DividerVerticalExampleShorthand = () => (
+  <div style={{ display: 'flex', justifyContent: 'center', height: '32px', alignItems: 'center' }}>
+    <Divider vertical />
```

Comment:
Should we support vertical Divider with content? Just curious how would it look like

Reply:
If this is the case I believe it would complicate the styles in some way, as we will need to "leave space" between the two parts of the divider for the text..

Reply:
It wasn't a requirement so far. If it's with content then it changes significantly, as it will be `| content |` instead of just `|`. While the horizontal one is still a one liner. Do you think we need to handle this case now or can it be left for when we really need it?

Reply:
If Teams needs doesn't need this, then it's okay.